openshift/template.yaml

```diff
@@ -77,15 +77,26 @@ objects:
                       name: worker
                       key: sentry-auth-token
                 - name: REPORT_BUCKET_NAME
-                  value: developer-analytics-audit-report
+                  valueFrom:
+                    secretKeyRef:
+                      name: developer-analytics-audit-report-s3
```

Comment:
Why do we need secret keys here? Becomes very difficult to change or modify later with a different bucket name by opening a PR for Vault. Please read it from an environment parameter as before. Ditto for others as well.

Reply:
It was decided to use more granular access permissions for each bucket having their own access keys.